### PR TITLE
Fix end scores not showing up at the end of multiplayer sessions

### LINKED ISSUE #100: [BUG] Multiplayer end screen doesn't report player scores
Honestly, I'm not sure if this is a bug, or a new feature that should be implemented. Regardless:

[//]: # (Describe what's going wrong:)
### What's going wrong

After completing a multiplayer match, the end screen should show the scores that the players got in the just-completed match. Those do not show up in OpenMBU.

Screenshot:
MBUFinalScreen-OpenMBU

[//]: # (Describe what should happen:)
### What should happen

After completing a multiplayer match, the end screen should show the scores that the players got in the just-completed match.

Screenshot from Xbox 360 (taken from https://www.youtube.com/watch?v=eN0JBRWITTU):
MBUFinalScreen-Xbox360

I'm not sure how relevant this is, since the old PC port of MBU was based on MBO and a lot of menus are different, but the scores also show up in the old port. This is how it looks:
MBUFinalScreen-OldPCPort

### PR DESCRIPTION
**Pull request by thearst3rd**
Fixes #100.

Note, there still seems to be an issue where the text is always white, rather than being colored black or orange when highlighted.

https://www.youtube.com/watch?v=DuXGb6rumew&t=323s

### CODE REVIEW

File: game/marble/client/scripts/game.cs [hunk 1]
```diff
@@ -939,7 +939,8 @@ function clientCmdSetGameState(%state, %data)
       // if we are in end state, write scores
       if (XBLiveIsStatsSessionActive() && (%state $= "end" || %state $= "wait"))
       {
-         if (%allowStats && $Client::currentGameCounts && %state $= "end")
+         //if (%allowStats && $Client::currentGameCounts && %state $= "end")
+         if (%state $= "end")
```
Comment (thearst3rd) [inline game/marble/client/scripts/game.cs:943]: I'm not 100% confident in this. We might want to figure out what those two parameters are and why they aren't being set properly. But this seems to work without causing any negative side effects (including singleplayer), so I did it for now.

Comment (HumanGamer) [reply game/marble/client/scripts/game.cs:943]: I would like to find out what those are for before merging this.

Comment (thearst3rd) [reply game/marble/client/scripts/game.cs]: I've reverted this change in favor of another change which enables the stats. This potentially has the downside of running extra xblive specific code which might print extra "command not found" things into the log, but everything seems to work fine including both singleplayer and multiplayer. I have no idea which change is better, so I'll leave that up to you. At least I think we know vaguely how those variables work, or at least why they weren't set before

**Comment by thearst3rd** [comment]
Waiting to merge this until the ending colors are fixed, will come after beta 1.11

**Pull request merged**